### PR TITLE
chore(ci): create fast feedback unit test profile

### PR DESCRIPTION
**Pull request by soonum**
This is done to get quick feedback to developpers in a Pull Request.

It tests shorint level with only three sets of parameters. Integer level is tested with only the default operations with two sets of parameters.

This profile will be automatically triggered on each push in a pull request.
Conversely the full suite of test will also be triggered automatically but once the review is approved.

### CODE REVIEW

**Review by IceTDrinker — APPROVED** [review]
Test approve

File: .github/workflows/aws_tfhe_fast_tests.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        run: |
+          BIG_TESTS_INSTANCE=TRUE FAST_TESTS=TRUE make test_integer_ci
+
+      - name: Run shortint multi-bit tests
+        run: |
+          BIG_TESTS_INSTANCE=TRUE FAST_TESTS=TRUE make test_shortint_multi_bit_ci
+
+      - name: Run integer multi-bit tests
+        run: |
+          BIG_TESTS_INSTANCE=TRUE FAST_TESTS=TRUE make test_integer_multi_bit_ci
```
Comment (IceTDrinker) [inline .github/workflows/aws_tfhe_fast_tests.yml:103]: we'll see if BIG_TESTS_INSTANCE thing holds for multi bit, but as we run less parameters it should be fine

**Review by IceTDrinker — APPROVED** [review]
Looks good !

**Comment by IceTDrinker** [comment]
hmm looks like it relaunches the fast tests ?

**Comment by soonum** [comment]
> hmm looks like it relaunches the fast tests ?

Yes it's expected since the fast tests is the test suite that is automatically launched on each push in a PR.

**Comment by IceTDrinker** [comment]
what I mean is what's the point of relaunching the fast test on approval since it's contained in the full test suite ?

**Comment by soonum** [comment]
You're right, I'll fix that right away.

**Comment by soonum** [comment]
That was a false impression. I've checked the Actions runs and the fast tests is not launched upon approval.
I'm merging as is.

**Review by IceTDrinker — APPROVED** [review]

**Pull request merged**